app/views/cases/index.html.erb

```diff
@@ -41,28 +37,9 @@
           <th>LMP</th> <!-- last menstrual period -->
           <th>Case Notes</th>
           <th></th>
+          <th></th>
         </tr>
-        <tr>
-          <td>202-345-6677</td>
-          <td>Milly G</td>
-          <td>17 weeks</td>
-          <td>(sample case notes. something or other. detail)</td>
-          <td><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></td>
-        </tr>
-        <tr>
-          <td>202-345-6677</td>
-          <td>Susan B</td>
-          <td>12 weeks</td>
-          <td>(sample case notes. something or other. detail)</td>
-          <td><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></td>
-        </tr>
-        <tr>
-          <td>202-345-6677</td>
-          <td>Patty W</td>
-          <td>9 weeks</td>
-          <td>(sample case notes. something or other. detail)</td>
-          <td><span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span></td>
-        </tr>
+        <%= render partial: 'table_content', locals: {cases: current_user.cases} %>
```

Comment:
Ahhhh I see what's up here. Sorry for commenting before understanding! If this is tied to `current_user` it's solid.